jvmctl/jvmctl/jvmctl.py

```diff
@@ -905,7 +905,7 @@ def post_config(node):
     }
     property_opts = fmt_properties(properties) + ' ' + fmt_properties(node.container.properties)
     jvm_opts = ['-Xmx' + node.config.get('jvm', 'HEAP_SIZE'),
-                '-XX:OnOutOfMemoryError=/usr/bin/jvmctl oomkill ' + node.name + ' %p']
+                '-XX:OnOutOfMemoryError=/usr/bin/jvmctl oomkill ' + node.name + ' %p' + '-Dlog4j2.formatMsgNoLookups=true' ]
```

Comment:
```suggestion
                '-XX:OnOutOfMemoryError=/usr/bin/jvmctl oomkill ' + node.name + ' %p', '-Dlog4j2.formatMsgNoLookups=true' ]
```
jvm_opts is a list so this needs to be a comma instead of a plus or it'll end up effectively generating

    -XX:OnOutOfMemoryError="/usr/bin/jvmctl oomkill myapp 1234-Dlog4j2.formatMsgNoLookups=true"

instead of

    -XX:OnOutOfMemoryError="/usr/bin/jvmctl oomkill myapp 1234" -Dlog4j2.formatMsgNoLookups=true